locksmith/src/utils/keyPricer.ts

```diff
@@ -36,26 +32,18 @@ export default class KeyPricer {
     const lock = await this.readOnlyEthereumService.getLock(
       Normalizer.ethereumAddress(lockAddress),
       network,
-      { fields: ['currencyContractAddress', 'currencySymbol', 'keyPrice'] }
+      { fields: ['currencyContractAddress', 'keyPrice'] }
```

Comment:
I want to make sure `getLock` actually only loads these 2 fields... but I suspect it may create a first request to get the version...